irohad/main/iroha_conf_loader.cpp

```diff
@@ -431,6 +446,18 @@ inline void JsonDeserializerImpl::getVal<IrohadConfig::DbConfig>(
       path, dest.maintenance_dbname, obj, config_members::MaintenanceDbName);
 }
 
+template <>
+inline void JsonDeserializerImpl::getVal<IrohadConfig::UtilityService>(
+    const std::string &path,
+    IrohadConfig::UtilityService &dest,
+    const rapidjson::Value &src) {
+  assert_fatal(src.IsObject(),
+               path + " utility service config top element must be an object.");
+  const auto obj = src.GetObject();
+  getValByKey(path, dest.ip, obj, config_members::Ip);
```

Comment:
> config_members::Ip

Can a domain name be passed instead of ip4 as value?

Reply:
it is the listening ip address, used to limit the network scope of this endpoint.